tests/gui/models/selection-state.spec.js

```diff
@@ -3,74 +3,63 @@
 const m = require('mochainon');
 const _ = require('lodash');
 const path = require('path');
-const angular = require('angular');
 const availableDrives = require('../../../lib/gui/models/available-drives');
-require('angular-mocks');
+const selectionState = require('../../../lib/gui/models/selection-state');
 
-describe('Browser: SelectionState', function() {
+describe('Browser: selectionState', function() {
 
-  beforeEach(angular.mock.module(
-    require('../../../lib/gui/models/selection-state')
-  ));
-
-  describe('SelectionStateModel', function() {
-
-    let SelectionStateModel;
-
-    beforeEach(angular.mock.inject(function(_SelectionStateModel_) {
-      SelectionStateModel = _SelectionStateModel_;
-    }));
+  describe('selectionState', function() {
```

Comment:
I think this `describe` block is redundant given that we're already saying `Browser: selectionState` above. Maybe we can remove it and get rid of one indentation level? :)